### PR TITLE
Adds the T-70 grenade launcher 

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Adds the T-70 grenade launcher and grenades to marine vendors with 

## Why It's Good For The Game
gives the marines a heavy support weapon.

## Changelog
:cl:
add: Adds the T-70 grenade launcher, and grenades to marine vendors
balance: Adds the T-70 grenade launchers and HEDPs to marine vendors.
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -18. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by Iamgoofball** [comment]
>>>You currently have a negative Fix/Feature pull request delta of -18.

**Comment by XSlayer300** [comment]
Right hand and left hand sprites on the mob are again, inverted. This time at the front and back. The sides are not covered as well. http://prntscr.com/t35bh2

Here's how it looks proper. http://prntscr.com/t35bld

File: code/modules/projectiles/gun_attachables.dm [hunk 1]
```diff
@@ -1075,6 +1075,20 @@ Defined in conflicts.dm of the #defines folder.
 	scatter_mod = 0
 	movement_acc_penalty_mod = 0
 
+/obj/item/attachable/stock/t70stock
+	name = "T-70 stock"
```
Comment (XSlayer300) [inline code/modules/projectiles/gun_attachables.dm:1079]: Add `\improper` before the stock?

**Comment by ilikesaying** [comment]
haha friendly fire go BOOM

**Comment by jroinc** [comment]
10 FULLY-LOADED 6-round grenade launchers, and 30 grenades PER VENDOR seems a bit strong.

Unload the GL's, and cut the number of GL's to 5, keep grenade amounts the same.

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-19). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**